resources/tools/index.md

```diff
@@ -0,0 +1,90 @@
+                        <p><a href="https://xmrchain.net/">xmrchain.net</a></p>
+                        <p><a href="https://moneroblocks.info">MoneroBlocks</a></p>
+                        <p><a href="https://www.exploremonero.com/">Explore Monero</a></p>
+                        <p><a href="https://monerovision.com">MoneroVision</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.address-generators %}</h2>
+                        <p><a href="https://moneroaddress.org/">moneroaddress.org</a></p>
+                        <p><a href="https://xmr.llcoins.net/">xmr.llcoins.net</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="left half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.network %}</h2>
+                        <p><a href="https://monerohash.com/nodes-distribution.html">monerohsah.com - {% t tools.monerohash-nodes %}</a></p>
+                        <p><a href="https://moneroblocks.info/stats">MoneroBlocks - {% t tools.moneroblocks-stats %}</a></p>
+                        <p><a href="https://nownodes.io/connect-xmr-node">NOWNodes - {% t tools.nownodes %}</a></p>
+                        <p><a href="https://getblock.io">GetBlock - {% t tools.getblockio %}</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.market %}</h2>
+                        <p><a href="https://www.backtestking.com/">Backtestking - {% t tools.backtestking %}</a></p>
+                        <p><a href="https://cryptofacile.io">cryptoFacile - {% t tools.cryptofacile %}</a></p>
+                        <p><a href="https://cryptoradar.co/buy-monero">Cryptoradar - {% t tools.cryptoradar %}</a></p>
+                        <p><a href="https://kryptocheck.de">KryptoCheck - {% t tools.kryptocheck %}</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="left half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.misc %}</h2>
+                        <p><a href="http://xmrtests.llcoins.net/checktx.html">xmrtests.llcoins.net - {% t tools.llcoins-checktx %}</a></p>
```

Comment:
If they don't have a TLS certificate we don't list them. Please remove `xmrtests.llcoins.net` and its related strings.